src/components/fpl/MatchResults.tsx

```
@@ -189,128 +209,159 @@ export default function MatchResults({
     <div className="space-y-4 lg:space-y-6">
+      {/* Manager Selection */}
+      {selectedManagerId && (
+        <div className="bg-theme-card rounded-md p-4 border-theme-border theme-transition mb-4">
+          <h4 className="text-lg font-bold text-theme-foreground mb-2 theme-transition">
+            Manager #{selectedManagerId} Statistics
+          </h4>
+          <button
+            onClick={() => setSelectedManagerId(null)}
+            className="text-sm px-3 py-1 bg-gray-200 dark:bg-gray-700 rounded-md hover:bg-gray-300 dark:hover:bg-gray-600 transition-colors theme-transition"
+          >
+            Clear Selection
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)